CHANGELOG.md

```diff
@@ -1,5 +1,62 @@
 # Change Log
 
+## [0.8.0](https://github.com/angelozerr/lsp4xml/milestone/9?closed=1) (July 23, 2019)
+
+### Enhancements
+
+ * Find definition for start/end tag element. See[#535](https://github.com/angelozerr/lsp4xml/issues/535)
+ * Improve Source information from completion. See[#529](https://github.com/angelozerr/lsp4xml/issues/529)
+ * `XSD CodeLens` references should be clickable. See[#490](https://github.com/angelozerr/lsp4xml/issues/490)
```

Comment:
Can we say "Add support for clickable XSD CodeLens" instead? I'm personally don't think it should say "should be clickable".